packages/graphql/src/contracts.js

```diff
@@ -245,88 +268,79 @@ export function setNetwork(net, customConfig) {
       ethGasStation: ['mainnet', 'rinkeby'].includes(net)
     })
   }
+}
+
+function setupWeb3(config, net) {
+  const provider = process.env.PROVIDER_URL
```

Comment:
nit: how about defining getEnvVar as a function outside of setupOriginProviders so that it can also be used here and in other functions in the future ?